.github/workflows/sync-from-snapshot-mainnet-paritydb-pruned.yml

```diff
@@ -7,20 +7,60 @@
 
 name: Sync from snapshot, Mainnet, ParityDB pruned
 on:
-  # At 03:00 on Wednesday and Saturday
+  # At 20:00 on Wednesday and Saturday
```

Comment:
You changed these quite a couple of times, which suggests they are not quite arbitrary – could you add a short comment explaining why the current times have been chosen as they have? Or is this semi-random after all?

Reply:
Those times correspond with db backups.